### DISCUSSION TITLE
Your Codewars account has been suspended

### DISCUSSION THREAD

**Discussion by maximmorenko** [codewars/codewars.com#3207] [Q&A] [answered]
Hello, I was banned today.
Your Codewars account has been suspended due to violating our terms.
Reason for removal: Plagiarism.

Plagiarism? What does it mean? What smart person will decide whether it is plagiarism or not? And most importantly, how does he determine this? If you think that people are not capable of writing the same solutions, then I feel sorry for you. For the same reason I was banned, you should ban 2/3 of Codewars accounts. I’ll tell you a secret, no matter how complex a person’s brain is, the number of solutions to one kata will always be less than the number of people who solved it. It doesn’t take much intelligence to understand that decisions will inevitably be repeated, does it? I am ready to answer for every kata I solve. And you? Are you ready to take responsibility for your decisions? kindergarten

**Comment by hobovsky** [comment, marked as answer]
Plagiarism means: copying code of other people and submitting as your own to get points. Which, after my brief review, did happen in case of this account.
No need to worry, the suspension is not permanent, you can see that _"[...]  the suspension ends on May 29, 2024"_. The invalidated solutions will stay invalidated tho. If you do not agree with the decision of a moderator about suspending the account, you can complain with an email to `info@codewars.com` and your case has a chance to be reviewed.

**Comment by maximmorenko** [comment]
I don't care about your worthless bans, you can reset my rank and my total score, unlike you, I have a different goal. Only knowledge and experience.
You will definitely be awarded a medal for your activities. Good luck)
